### PR TITLE
feat!: use one configuration for all backends

### PR DESCRIPTION
**Pull request by Ian2012**
**Description:** This PR updates the configuration to only use 1 setting entry for both backends.

This is a proposal, and depends on: https://github.com/openedx/event-tracking/pull/273

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: event_routing_backends/settings/common.py [hunk 1]
```diff
@@ -173,26 +173,34 @@ def plugin_settings(settings):
...
     ]
 
-    settings.EVENT_BUS_TRACKING_LOGS = set(allowed_xapi_events + allowed_caliper_events)
+    allowed_events = set(allowed_xapi_events + allowed_caliper_events)
+
+    settings.EVENT_BUS_TRACKING_LOGS = allowed_events
 
     settings.EVENT_TRACKING_BACKENDS.update({
-        'xapi': {
+        'events': {
```
Comment (ziafazal) [inline event_routing_backends/settings/common.py:181]: I think instead of `events` we should have `event_transformer` or `event_routing`.

**Review by ziafazal — COMMENTED** [review]
@Ian2012 I'm fine with using single backend how name of that backend should not be just `events` I have suggested couple of names in other comment.

File: event_routing_backends/settings/common.py [hunk 2]
```diff
@@ -173,18 +173,20 @@ def plugin_settings(settings):
         'edx.course.grade.now_failed'
     ]
 
-    settings.EVENT_BUS_TRACKING_LOGS = set(allowed_xapi_events + allowed_caliper_events)
+    allowed_events = set(allowed_xapi_events + allowed_caliper_events)
+
+    settings.EVENT_BUS_TRACKING_LOGS = allowed_events
```
Comment (bmtcril) [inline event_routing_backends/settings/common.py:178]: Can you document what this setting does here? It's not used anywhere else in the repo.

**Review by bmtcril — APPROVED** [review]
Just had the one request, otherwise it looks good to me

File: event_routing_backends/settings/common.py (hunk 2 above)
Comment (Ian2012) [reply event_routing_backends/settings/common.py:181]: is the current change enough?

Comment (bmtcril) [reply event_routing_backends/settings/common.py:181]: That's better, thanks

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/event-routing-backends&entry.867055334=https://github.com/openedx/event-routing-backends/pull/411&entry.1484655318=https://github.com/Ian2012&entry.752974735=2024-04-01+20:32&entry.1917517419=2024-04-02+20:15&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**